File: template.ts
import { strict as assert } from "assert";
import { readFileSync } from "fs";

const parse = function (file: string): string[] {
  return readFileSync(file, "utf-8")
    .trim()
    .split("\n");
};

const solve1 = function (file: string): number {
  const lines = parse(file);

  return 0;
};

const solve2 = function (file: string): number {
  const lines = parse(file); 

  return 0;
};

//assert(solve1("./example.txt") === 3);
console.log(solve1("./input.txt"));

//assert(solve2("./example2.txt") === 2);
console.log(solve2("./input.txt"));

File: template.py
#!/usr/bin/env python3

import sys


def get_lines(filename):
    return [l.strip() for l in open(filename, "r")]


def part1(filename):
    lines = get_lines(filename)

    return 0


def part2(filename):
    lines = get_lines(filename)

    return 0


def main():
    assert part1("example.txt") == 0
    print(part1("../input/2018/day1.txt"))

    assert part2("example.txt") == 0
    print(part2("../input/2018/day1.txt"))


if __name__ == "__main__":
    sys.exit(main())
